crates/bevy_internal/src/prelude.rs

```diff
@@ -1,7 +1,7 @@
 #[doc(hidden)]
 pub use crate::{
-    app::prelude::*, core::prelude::*, ecs::prelude::*, hierarchy::prelude::*, input::prelude::*,
-    log::prelude::*, math::prelude::*, reflect::prelude::*, time::prelude::*,
+    app::prelude::*, core::prelude::*, ecs::prelude::*, hierarchy::prelude::*, image::prelude::*,
+    input::prelude::*, log::prelude::*, math::prelude::*, reflect::prelude::*, time::prelude::*,
```

Comment:
```suggestion
    app::prelude::*, core::prelude::*, ecs::prelude::*, hierarchy::prelude::*, input::prelude::*,
    log::prelude::*, math::prelude::*, reflect::prelude::*, time::prelude::*,
```

and you'll need to add

```
#[doc(hidden)]
#[cfg(feature = "bevy_image")]
pub use crate::image::prelude::*;
```